### DISCUSSION TITLE
I set "true", but the click menu does not collapse and expand (invalid click).

### DISCUSSION THREAD

**Discussion by heihahehe** [thuliteio/doks#1154] [Q&A]
![1229225159](https://github.com/gethyas/doks/assets/152707762/66480d7b-659a-4475-94e2-f377cf6cfd57)

**Comment by h-enk** [comment]
Please share your repo – I could look into your issues coming Wednesday

**Reply by heihahehe** [reply]
Sorry, I didn't upload to github, the problem is that the multilingual buttons don't open, the small menu doesn't open, and the docs directory doesn't expand.

**Reply by h-enk** [reply]
I can't replicate your issues and help you properly without a repository

**Reply by heihahehe** [reply]
https://github.com/heihahehe/docs_ss

**Reply by heihahehe** [reply]
Thank you, I have uploaded it to github.

**Reply by heihahehe** [reply]
I found the cause of the problem, when I "flexSearch = false", the problem occurs.

**Reply by h-enk** [reply]
Then:

- Load Bootstrap JavaScript (set `bootstrapJavascript = true`) — https://github.com/gethyas/doks/blob/main/config/_default/params.toml#L38

- Copy `node_modules/@hyas/doks-core/assets/js/bootstrap.js` to `./assets/js/bootstrap.js` and import what you need — https://github.com/gethyas/doks-core/blob/main/assets/js/bootstrap.js
